pkg/kubectl/cmd/config/current_context.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+type CurrentContextOptions struct {
+	ConfigAccess ConfigAccess
+}
+
+const (
+	current_context_long    = `Displays the current-context`
+	current_context_example = `# Display the current-context
+$ kubectl config current-context`
+)
+
+func NewCmdConfigCurrentContext(out io.Writer, ConfigAccess ConfigAccess) *cobra.Command {
+	options := &CurrentContextOptions{ConfigAccess: ConfigAccess}
+
+	cmd := &cobra.Command{
+		Use:     "current-context",
+		Short:   "Displays the current-context",
+		Long:    current_context_long,
+		Example: current_context_example,
+		Run: func(cmd *cobra.Command, args []string) {
+			config, err := options.ConfigAccess.GetStartingConfig()
+			if err != nil {
+				fmt.Fprintf(out, "%v\n", err)
+			}
+
+			if config.CurrentContext == "" {
+				fmt.Fprintf(out, "current-context is not set")
```

Comment:
I would suggest printing this message to stderr instead of stdout, so that running the command in scripts gets the empty string if no context is set. @janetkuo do you know if there are other commands that do that?